### ISSUE TITLE
 Liquid Exception: undefined method `each' for nil:NilClass

### ISSUE THREAD

**Issue by bauerjj** [ayastreb/jekyll-maps#26]
Trying to follow the instructions [here](https://github.com/ayastreb/jekyll-maps#data-source):

I can get one marker working with the map displaying fine, however if I attempt to load from the described `places.yml` file, then I get an error as seen in the title

```
{% google_map src="_data/places.yml" %}
```
```
# _data/places.yaml
  - title: "Paris"
    url: "http://google.fr"
    location:
      latitude: 48.8587741
      longitude: 2.2074741

  - title: "Madrid"
    url: "http://google.es"
    location:
      latitude: 40.4378698
      longitude: -3.8196204
```

stack trace:
```
C:/tools/ruby23/lib/ruby/gems/2.3.0/gems/jekyll-maps-2.0.2/lib/jekyll-maps/location_finder.rb:14:in `find': undefined method `each' for nil:NilClass (NoMethodError)
from C:/tools/ruby23/lib/ruby/gems/2.3.0/gems/jekyll-maps-2.0.2/lib/jekyll-maps/google_map_tag.rb:15:in `render' from C:/tools/ruby23/lib/ruby/gems/2.3.0/gems/liquid-3.0.6/lib/liquid/block.rb:151:in `render_token'
        from C:/tools/ruby23/lib/ruby/gems/2.3.0/gems/liquid-3.0.6/lib/liquid/profiler/hooks.rb:5:in `block in render_token_with_profiling'
        from C:/tools/ruby23/lib/ruby/gems/2.3.0/gems/liquid-3.0.6/lib/liquid/profiler.rb:80:in `profile_token_render'
....
....
````

**Comment by ayastreb** [comment]
Hi @bauerjj 
sorry, was a bug there - fixed now in 2.0.3.
thanks for reporting it!

**Comment by bauerjj** [comment]
it works!
